data/test_yaml_stories/stories_hybrid_e2e.yml

```diff
@@ -0,0 +1,15 @@
+stories:
+- story: My hybrid End-to-End story
```

Comment:
Question - in the previous test file `stories_e2e.yml`, the `Hybrid` stories are called `mixed`. Do we plan to use both terms or me we can agree on one?

Reply:
hm, I don't know, for now they are not terms, but rather used in the literal meaning of these words